packages/dos-donts-block/src/helpers/getPlugins.ts

```diff
@@ -0,0 +1,54 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { AppBridgeBlock } from '@frontify/app-bridge';
+import {
+    AlignCenterPlugin,
+    AlignJustifyPlugin,
+    AlignLeftPlugin,
+    AlignRightPlugin,
+    BoldPlugin,
+    CheckboxListPlugin,
+    CodePlugin,
+    EmojiPlugin,
+    ItalicPlugin,
+    OrderedListPlugin,
+    ParagraphPlugin,
+    PluginComposer,
+    ResetFormattingPlugin,
+    SoftBreakPlugin,
+    StrikethroughPlugin,
+    TextStylePlugin,
+    UnderlinePlugin,
+    UnorderedListPlugin,
+} from '@frontify/fondue';
+import { ButtonPlugin, LinkPlugin } from '@frontify/guideline-blocks-shared';
+
+export const getPlugins = (appBridge: AppBridgeBlock) => {
```

Comment:
Probably it makes sense to but some (Titile, description [this one?]) Plugin Settings into the shared, so we can reuse them everywhere

Reply:
Created a `pluginPresets` library in the shared folder. What do you think?

Reply:
Cool! Thanks.